### DISCUSSION TITLE
Paramount Network (España/Spain)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#346] [Show and tell]
### Channel Name

Paramount Network

### Channel ID

ParamountNetworkSpain.es

### Stream URL

http://154.62.74.11:1935/live/paramount.smil/playlist.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
http://154.62.74.11:1935/live/paramount.smil/playlist.m3u8 (offline)
<img width="415" alt="image" src="https://user-images.githubusercontent.com/7253922/232225895-e7a2f8a4-9a43-41f3-b9fc-b9d53317833e.png">
